README.md

```diff
@@ -71,14 +71,24 @@ Control over the fpm daemon's state; set these to `stopped` and `false` if you w
 
 The handler restarts PHP-FPM by default. Setting the value to `reloaded` will reload the service, intead of restarting it.
 
-    php_fpm_listen: "127.0.0.1:9000"
-    php_fpm_listen_allowed_clients: "127.0.0.1"
-    php_fpm_pm_max_children: 50
-    php_fpm_pm_start_servers: 5
-    php_fpm_pm_min_spare_servers: 5
-    php_fpm_pm_max_spare_servers: 5
-
-Specific settings inside the default `www.conf` PHP-FPM pool. If you'd like to manage additional settings, you can do so either by replacing the file with your own template or using `lineinfile` like this role does inside `tasks/configure-fpm.yml`.
+    php_fpm_pools
```

Comment:
It looks like the formatting here got screwed up a little bit. This should be displaying the same as it is in defaults.yml—right now it looks like `php_fpm_pools` is just a bare variable... and the pool below is just floating in space on its own.

Maybe print the entire `php_fpm_pools` var just like it is in defaults, then drop the documentation line about what this is below the code block.